src/features/tokens/TokenListModal.tsx

```diff
@@ -81,6 +82,15 @@ export function TokenList({
       const hasRoute = hasTokenRoute(originChainId, destinationChainId, t.address, tokenRoutes);
       if (!q) return hasRoute;
       else
+        if(t.type === TokenType.collateral) {
+          return (
+            hasRoute &&
+            (t.name.toLowerCase().includes(q) ||
+              t.symbol.toLowerCase().includes(q) ||
+              t.address.toLowerCase().includes(q)) ||
+              t.isERC721
+          );
```

Comment:
It seems like this shows ERC721 tokens in search results regardless of if they match the search query. Was that intentional?